component/pom.xml

```diff
@@ -24,14 +24,13 @@
     <parent>
         <groupId>org.wso2.extension.siddhi.gpl.execution.geo</groupId>
         <artifactId>siddhi-gpl-execution-geo-parent</artifactId>
-        <version>3.1.3-SNAPSHOT</version>
+        <version>4.0.0</version>
```

Comment:
@suhothayan Fixed the version.  
"-parent" in the artifactId is just to refer to the parent pom.